apps/civicsignalblog/tsconfig.json

```diff
@@ -20,7 +20,13 @@
       "@/commons-ui/next/*": ["../../packages/commons-ui-next/src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "src/payload/components/BeforeDashboard/index.jsx",
+    "src/payload/components/BeforeDashboard/index.jsx"
```

Comment:
What does including/not including them here actually do?

Reply:
I don't even know how that got there 🤦 , must've been automatically added, I've removed it !